demdata_cms/src/taccsite_cms/settings_custom.py

```diff
@@ -1,24 +1,15 @@
 # CUSTOM SETTINGS VALUES.
 # TACC WMA CMS SITE:
 # *.DEMOCTRATIZING-SITE.TACC.UTEXAS.EDU
-# https://democratizingdata.ai/
 
 ########################
-# TACC: PORTAL
-########################
-
-# Does this CMS site have a portal (default value: True)?
-INCLUDES_CORE_PORTAL = False
-INCLUDES_PORTAL_NAV = False
-INCLUDES_SEARCH_BAR = False
```

Comment:
**Woah, Pardner. Deleting these?** Nope. Just moving them down further, to match Core-CMS order.